gradle/libs.versions.toml

```diff
@@ -207,3 +207,4 @@ aws-sdk-netty-nio = { module = "software.amazon.awssdk:netty-nio-client" }
 
 # vulnerability
 cosv4k = { module = "com.saveourtool.cosv4k:cosv4k", version.ref = "cosv4k" }
+
```

Comment:
just an empty line?

Reply:
in local deployment we need to call `kotlinYarnUpdate` task to make `yarn.lock` be upgraded

however, CI fails and cant recognize the `yarn` updates

Thus, I change `lib.toml` in aim to trigger CI task for updating the `yarn.lock`, but diff is empty 😐